webpack.config.js

```diff
@@ -6,7 +6,7 @@ const webpack = require('webpack');
 const CleanObsoleteChunksPlugin = require('webpack-clean-obsolete-chunks');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
-
```

Comment:
Please keep this new line. This way it is easier to distinguish between imports and other code.

Reply:
i thought, that this would be fixed automatically when merging, because there are no conflicts. but i'll do the rebase.